site-src/v1alpha2/guides/traffic-splitting.md

```diff
@@ -1,9 +1,5 @@
 # HTTP traffic splitting
 
-!!! danger
-    This page has not been updated for v1alpha2 yet.
-
-
 The [HTTPRoute resource](/v1alpha2/api-types/httproute) allows you to specify weights to shift
 traffic between different backends. This is useful for splitting traffic during
 rollouts, canarying changes, or for emergencies. The HTTPRoute
```

Comment:
Can't comment on the line below this, but `forwardTo` references are out of date.

Reply:
2250888077cad438b0ce99f3242eae491ff024c0